<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>371</width>
     <height>238</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hzlLocation">
    <item>
     <layout class="QVBoxLayout" name="vtlLocation">
      <item>
       <widget class="QLCDNumber" name="lcd_time">
        <property name="digitCount">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_date">
        <property name="digitCount">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_longitude">
        <property name="digitCount">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_latitude">
        <property name="digitCount">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cboLocation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cboSkyObjects">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>371</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hzlSkyObjects">
    <item>
     <layout class="QVBoxLayout" name="vtlSkyObjects">
      <item>
       <widget class="QLabel" name="lblRektaszension">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RA [deg - h:m:s]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="tedRektaszension">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Rektaszension HH:MM:SS</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDeclination">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DEC [deg - grad:min:sec]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="tedDeklination">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAltitude">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ALT [deg]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="tedAltitude">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string extracomment="Azimut"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAzimut">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AZ [deg]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="tedAzimut">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string extracomment="Azimut"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdDistance">
        <property name="digitCount">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAboveHorizon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;green if object is visible&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>ABOVE HORIZON</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
